components/05-pages/landing-pages/landing-pages.stories.js

```diff
@@ -3,12 +3,15 @@ import { storiesOf } from '@storybook/react';
 import { hrefTo } from '@storybook/addon-links';
 import { useEffect } from '@storybook/client-api';
 
-import { globalData } from '../../../.storybook/globalData';
-
 import mainMenu from '../../02-molecules/menus/main-menu/main-menu';
 
 import home from './home.twig';
 
+import mainMenuData from '../../02-molecules/menus/main-menu/main-menu.yml';
+import breadcrumbData from '../../02-molecules/menus/breadcrumbs/breadcrumbs.yml';
+import socialMenuData from '../../02-molecules/menus/social/social-menu.yml';
+import footerMenuData from '../../02-molecules/menus/inline/inline-menu.yml';
```

Comment:
This is an example where I think a single `global-data.yml` file would be preferable.

We could just:
```
import globalData from '../../globalData.yml';
import homePageData from '../../landing-page.yml';
```
and then
`<div dangerouslySetInnerHTML={{ __html: home({ ...globalData, ...homePageData }) }} />`
To accomplish the same thing.

This is obviously a developer preference, both work just fine.

Reply:
Let's stick with component data for now, and then see how we like it going into real projects. I'm fine moving to global as well if it makes more sense over time